HesoyamHospital/Backend/Service/AppointmentMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Backend.Model.PatientModel;
+using Backend.Model.UserModel;
+using Backend.Util;
+
+namespace Backend.Service
+{
+    public static class AppointmentMapper
+    {
+        public const long AppointmentDurationMinutes = 30;
+        public static Appointment AppointmentDtoToAppointment(AppointmentDTO dto)
+        {
+            Appointment appointment = new Appointment();
+            Doctor doctorInAppointment = AppResources.getInstance().doctorRepository.GetByID(dto.DoctorId);
+            appointment.DoctorInAppointment = doctorInAppointment;
+            appointment.Canceled = false;
+            appointment.AbleToFillOutSurvey = true;
+            appointment.AppointmentType = AppointmentType.checkup;
+            appointment.Room = GetRoom(doctorInAppointment);
+            appointment.TimeInterval = new TimeInterval();
```

Comment:
Use TimeInterval class constructor to create an object which can be passed to appointment constructor afterwards.